diff --git a/setup_nmssmtools.sh b/setup_nmssmtools.sh
index 6b6a60d..eeee9d2 100755
--- a/setup_nmssmtools.sh
+++ b/setup_nmssmtools.sh
@@ -78,7 +78,7 @@ copy_file() {
 }
 
 copy_files() {
-    copy_file ${BASEDIR}/nmProcessSpec.f ${nmssmtools_dir}/main/nmProcessSpec.f
+    copy_file ${BASEDIR}/src/nmProcessSpec.f ${nmssmtools_dir}/main/nmProcessSpec.f
     copy_file ${BASEDIR}/Makefile.nmssmtools ${nmssmtools_dir}/main/Makefile
 }
 
